content/docs/introduction/media.md

```diff
@@ -12,6 +12,13 @@ Resources on the Internet helpful to get started with Prometheus.
 * This site has its own [blog](http://prometheus.io/blog/).
 * [SoundCloud's blog post announcing Prometheus](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud) – a more elaborate overview than the one given on this site.
 * The [monitoring series](http://www.boxever.com/tag/monitoring) on Boxever's tech blog.
+* The [Robust Perception blog](http://www.robustperception.io/blog/).
```

Comment:
You want `/tag/prometheus` here. Probably best to put it above Boxever, as that has fewer posts and they're getting outdated.